### PR TITLE
update: README.md

### PR DESCRIPTION
**Pull request by anaptfox**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+
+Run:
+
+```bash
+go build -o algolia cmd/algolia/main.go`
+```
+
+### How it works
+
+Under the hood, the connector uses [Algolia's Go Library](https://www.algolia.com/doc/api-client/getting-started/install/go/?client=go) to send data to Algoliga.
```
Comment (hariso) [inline README.md:17]: Typo in Algoliga.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,29 @@
...
+|----------------------|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------------|
+| `APIKey`            | destination | The [API key](https://www.algolia.com/doc/guides/security/api-keys/) for Algolia.                                                                                                                      | true     |               |
+| `ApplicationID`              | destination | The [Application ID](https://www.algolia.com/doc/guides/security/api-keys/) for Algolia.                                                                                                                           | true     |               |
+| `IndexName`         | destination | The Algolia index where records get written into.                                                                          |  true   |               |
```
Comment (hariso) [inline README.md:29]: It would be good if we move to camel case, since that's the convention other connectors use too.

File: README.md (hunk 1 above)
Comment (anaptfox) [inline README.md:17]:
```suggestion
Under the hood, the connector uses [Algolia's Go Library](https://www.algolia.com/doc/api-client/getting-started/install/go/?client=go) to send data to Algolia.
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,29 @@
...
+
+### How it works
+
+Under the hood, the connector uses [Algolia's Go Library](https://www.algolia.com/doc/api-client/getting-started/install/go/?client=go) to send data to Algoliga.
+
+To learn more see: [How to build a Conduit Connector](https://www.conduit.io/guides/build-a-conduit-connecto)
+
+### Configuration
+
+There's no global, connector configuration. Each connector instance is configured separately.
```
Comment (anaptfox) [inline README.md:23]:
```suggestion
```

**Review by simonl2002 — APPROVED** [review]
lgtm

**Pull request merged**